AdyenDropIn/Presentation/WrapperViewController.swift

```diff
@@ -36,22 +36,30 @@ internal final class WrapperViewController: UIViewController {
         return viewController?.children.contains(where: { heirarchyRequiresKeyboardInput(viewController: $0) }) ?? false
     }
 
-    internal func updateFrame(keyboardRect: CGRect, animated: Bool) {
+    internal func updateFrame(keyboardRect: CGRect) {
         guard let view = child.viewIfLoaded, let window = UIApplication.shared.keyWindow else { return }
-        view.setNeedsLayout()
         let finalFrame = child.finalPresentationFrame(in: window, keyboardRect: keyboardRect)
-        view.layer.removeAllAnimations()
-        UIView.animate(withDuration: animated ? 0.35 : 0.0,
-                       delay: 0.0,
-                       options: [.curveLinear],
-                       animations: { view.frame = finalFrame },
-                       completion: nil)
+        topConstraint?.constant = finalFrame.origin.y
+        view.setNeedsLayout()
+        view.layoutIfNeeded()
```

Comment:
i don't think we need to call both methods here. `layoutIfNeeded` forces the layout updates immediately, and it is synchronous. `setNeedsLayout` waits for the next layout update cycle.
`layoutIfNeeded` shouldBeEnough here

Reply:
I tried it, its doesn't update the layout immediately unfortunately like you say, `view.layoutIfNeeded()` is unfortunately necessary.

Reply:
do you mean it did not work with only `layoutIfNeeded()`? seems like that method is the only one should be necessary

Reply:
removed `setNeedsLayout`